duckduckgo/Android
# Switch from using the favicon service to obtaining favicons directly from the website

Task/Issue URL: https://app.asana.com/0/488551667048375/1182893760422787
Fixes https://github.com/duckduckgo/Android/issues/877

**Description**:
There are a few places in the app where we use favicons:

1. Bookmarks (when viewing bookmarks screen)
1. Tabs switcher (when viewing tab switcher)
1. Fireproof websites (when viewing settings for Fireproofed websites)
1. Home screen shortcuts (when creating new home screen shortcut)

With this PR, we switch from using the favicon service to obtaining favicons directly from the website.


**Steps to test this PR**:
1. Check all places where favicons are used in the app to ensure they obtain the favicon directly from the website

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
